mmcv/runner/base_module.py

```diff
@@ -38,12 +63,38 @@ def is_init(self):
 
     def init_weights(self):
         """Initialize the weights."""
-        from ..cnn import initialize
 
+        # judge if it is Topmost module
```

Comment:
- judge -> check
- Topmost -> the topmost (actually I prefer "top-level", top can mean "most" in this case)